plugins/plugin-babel/test/plugin-babel.test.js

```diff
@@ -0,0 +1,117 @@
+const plugin = require('../plugin.js');
+const babel = require('@babel/core');
+const workerpool = require('workerpool');
+
+jest.mock('@babel/core');
```

Comment:
I believe this has to come before anything you import where you’re mocking this, right? (maybe you want to run the real `babel` in this test, but I think this would need to come before `require('../plugin.js')`, right?)

Reply:
Or are you loading this just to be able to keep track of the babel calls, without actually mocking it?

Reply:
Changed. 

Thanks for raising this. I believe `jest.mock` is hoisted to the top to support es import. 

> If you're using ES module imports then you'll normally be inclined to put your import statements at the top of the test file. But often you need to instruct Jest to use a mock before modules use it. For this reason, Jest will automatically hoist jest.mock calls to the top of the module (before any imports). 

But I am not 100% sure about this(no clear details found), whether it only triggers when there's `import` keyword, though I confirm it works in my mac.